### DISCUSSION TITLE
网页查询

### DISCUSSION THREAD

**Discussion by LotusRootMonkey** [chatchat-space/Langchain-Chatchat#3986] [General]
你好，请教一下
kimi.moonshot.cn，向kimi提问，它可以做到浏览网页，然后给出回答，并给出回答的出处来自哪个网址
现在我有一个，个人的网站，网站是个博客（内网），有很多文章，Langchain-Chatchat 能否做到让模型浏览我自己的内网网站的所有网页，给出回复并提供答案出自哪个网页

**Comment by lr13953301387** [comment]
不能，得自己实现

**Comment by zRzRzRzRzRzRzR** [comment]
不能
